python/helpers/requirements.txt

```
@@ -7,6 +7,7 @@ plette==2.1.0
 poetry==1.8.3
 # TODO: Replace 3p package `toml` with 3.11's new stdlib `tomllib` once we drop support for Python 3.10.
 toml==0.10.2
+uv==0.3.1
```

Though these guys ship 1 or 2 versions a week so this will go stale really quickly.
```suggestion
uv==0.4.9
```